### PR TITLE
nf-test ngsbits/samplegender

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/ngsbits/samplegender/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,49 @@
+// TODO nf-core: Once you have added the required tests, please run the following command to build this file:
+// nf-core modules test ngsbits/samplegender
```
Comment (FriederikeHanssen) [inline modules/nf-core/ngsbits/samplegender/tests/main.nf.test:2]:
```suggestion
```

Comment (maxulysse) [inline modules/nf-core/ngsbits/samplegender/tests/main.nf.test:2]:
```suggestion
```

Comment (nvnieuwk) [reply modules/nf-core/ngsbits/samplegender/tests/main.nf.test:2]: Thanks!

**Review by maxulysse — APPROVED** [review]

**Review by FriederikeHanssen — APPROVED** [review]

File: modules/nf-core/ngsbits/samplegender/tests/main.nf.test (hunk 1 above)
Comment (nvnieuwk) [reply modules/nf-core/ngsbits/samplegender/tests/main.nf.test]: @FriederikeHanssen was first :wink:

**Comment by nvnieuwk** [comment]
Thanks both!

**Pull request merged**